js/gateio.js

```diff
@@ -2517,7 +2517,7 @@ module.exports = class gateio extends Exchange {
             'average': undefined,
             'amount': amount,
             'cost': cost,
-            'filled': undefined,
+            'filled': this.safeFloat2 (order, 'amount', 'size') - this.safeFloat (order, 'left'),
```

Comment:
This is not necessary, since it's done in `safeOrder`, here:

- https://github.com/ccxt/ccxt/blob/master/js/gateio.js#L2503
- https://github.com/ccxt/ccxt/blob/master/js/base/Exchange.js#L1987
- https://github.com/ccxt/ccxt/blob/master/python/ccxt/base/exchange.py#L2615
- https://github.com/ccxt/ccxt/blob/master/php/Exchange.php#L3473

Reply:
Let me know if you have issues with that. The `filled` value should come out properly resolved from the order parser.

Reply:
Ah, thank you. Then the problem that I have must occur somewhere else, I'll close it for now and let you know if I found the problem and it's on the CCXT side.

Also: Not sure if you got notified about this my comment here since its a closed PR: https://github.com/ccxt/ccxt/pull/10521

Reply:
Yep, saw your comment, I think @frosty00 will reply there asap.